etc/init/rush-standalone.conf

```diff
@@ -0,0 +1,10 @@
+# auto start rush service if rush is installed standalone
+
+description "rush-standalone"
+
+start on filesystem or runlevel [2345]
+stop on shutdown
+
+script
+  [ ! -f /etc/init/spinnaker.conf ] && { start rush; }
```

Comment:
@dpeterka 
After playing with this and digging in, I understand more about what's going on with upstart so can comment on this.

Why is this needed as opposed to having rush (and other microservices) always start themselves?
stop / start spinnaker can iterate over the microservices as higher level orchestration for start/stop the whole collection.

Or, if not, then this attempt to start only if spinnaker isnt in there could go in the normal rush.conf, couldnt it?

The rush script (or here) needs to guard for cassandra being ready or rush is going to die. The spinnaker.conf also ensures that the log directory exists -- the rush.conf might as well be doing that because it wont restart if you delete /var/log/spinnaker/rush (which I independently discovered while testing this yesterday). So the spinnaker start isnt offering anything not already needed here.

In the rush config, I see it sets HOME to /home/spinnaker. This is potentially problematic because spinnaker HOME is not always /home/spinnaker (e.g. we have deployments where it wont be). Is there  a way to specify it as ~spinnaker? I'm not sure how important it is because there is nothing in /home/spinnaker anyway, unless you want to put a rush-local.yml (or spinnaker-local.yml) there.

When I modify the rush-standalone.conf to wait for cassandra as a pre-start script then this PR seems to be working ok technically. But it feels clunky with the standalone.conf, which also brings along another way to start it (which probably wont even work if you have a spinnaker installation, even if you arent running it). Seems like the special case "standaloneness" can be removed with the above changes.

Thoughts?